test/functionalTest/cases/4555_jexlexpression_in_subject_condition/jexl_basic_sum_eq3_attrs_in_update.test

```diff
@@ -0,0 +1,203 @@
+# Copyright 2025 Telefonica Investigacion y Desarrollo, S.A.U
```

Comment:
I'd suggest to add a variant of this test in a new .test in which only one of the attributes comes in the notification (e.g. A) and the other is an existing one in the entity (e.g. B).